src/common/Logging/Log.cpp

```diff
@@ -356,8 +364,14 @@ Log* Log::instance()
     return &instance;
 }
 
-void Log::Initialize()
+void Log::Initialize(Acore::Asio::IoContext* ioContext)
 {
+    if (ioContext)
+    {
+        _ioContext = ioContext;
+        _strand = new Acore::Asio::Strand(*ioContext);
```

Comment:
It's generally a C++ code smell to add a `new` without a `delete`. Consider whether this needs proper deallocation (perhaps in a dtor).

Reply:
This isnt undefine behavior

Reply:
I'm not saying this is undefined behavior, I'm saying that there is a risk of memory leak here.

Reply:
dumped in a delete